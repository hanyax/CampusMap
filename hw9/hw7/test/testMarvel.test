# Load the test small graph and find the paths for:

LoadGraph marvel marvel.tsv

FindPath marvel IRON_MAN/TONY_STARK_ HAWK

FindPath marvel IRON_MAN/TONY_STARK_ Batman

FindPath marvel Superman Batman
