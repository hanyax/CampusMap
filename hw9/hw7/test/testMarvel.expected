# Load the test small graph and find the paths for:

loaded graph marvel

path from IRON MAN/TONY STARK  to HAWK:
IRON MAN/TONY STARK  to HAWK with weight 0.003
total cost: 0.003

unknown character Batman

unknown character Superman Batman
